components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -329,9 +373,9 @@ abstract class AbstractFetchDownloadService : Service() {
 
     /**
      * Informs [mozilla.components.feature.downloads.manager.FetchDownloadManager] that a download
-     * has been completed.
+     * is no longer in progress due to being paused, completed, or failed
      */
-    private fun sendDownloadCompleteBroadcast(downloadID: Long, status: DownloadJobStatus) {
+    private fun sendDownloadNotInProgress(downloadID: Long, status: DownloadJobStatus) {
```

Comment:
Perhaps `sendDownloadFinished` is better? I know this name is ugly but I want to be clear about what's going on here.